packages/privatebin/default.nix

```diff
@@ -0,0 +1,14 @@
+{ pkgs ? import <nixpkgs> {}, mkDerivation ? pkgs.stdenv.mkDerivation, fetchFromGitHub ? pkgs.fetchFromGitHub }:
+
+mkDerivation {
+  name = "privatebin";
+  src = fetchFromGitHub {
+    owner = "m1cr0man";
```

Comment:
can we link this to the pr/issue outlineing why we are using @m1cr0man fork

Reply:
https://github.com/PrivateBin/PrivateBin/pull/553 this will be merged soon and I will undo this

Reply:
https://github.com/PrivateBin/PrivateBin/pull/553 this the one I'm assuming? If so I'll pop it in the message on this PR.

Reply:
Linked above also.